tilecloud_chain/generate.py

```diff
@@ -370,15 +349,14 @@ def _get_tilestore_and_message_for_layer(self, layer, gene):
                     params=params,
                 ),),
                 headers=layer['headers'],
-            ), "Get tile from WMS")
+            )
         elif layer['type'] == 'mapnik':  # pragma: no cover
             from tilecloud.store.mapnik_ import MapnikTileStore
             from tilecloud_chain.mapnik_ import MapnikDropActionTileStore
 
             grid = gene.get_grid(layer['grid'])
             if layer['output_format'] == 'grid':
-                self.count_tiles = gene.counter()
```

Comment:
If it is left in, having a mapnik grid layer in the config file will break the counter for all other layers (it will count tiles before the splitter can do its job)

This does mean mapnik grid layers can't count their generated tiles correctly, but as I mentioned above, that seemed better than all other layers not counting theirs